Jenkinsfile

```
@@ -104,7 +104,7 @@ pipeline {
                     steps {
                         container('cuda-10') {
                             dir("build-cuda") {
-                                sh "cmake -DAUTOPAS_ENABLE_CUDA=ON -DCCACHE=ON .."
+                                sh "cmake -DAUTOPAS_USE_TEST_GLOB=On -DAUTOPAS_ENABLE_CUDA=ON -DCCACHE=ON .."
```

```suggestion
                                sh "cmake -DAUTOPAS_USE_TEST_GLOB=ON -DAUTOPAS_ENABLE_CUDA=ON -DCCACHE=ON .."
```
This triggers me way more than it should ...
Btw you spelled all new flags that way.